app/models/event/synchronized_editor.rb

```diff
@@ -66,9 +64,19 @@ def self.create_for_editor_change(event, user, programming_group)
     )
   end
 
+  def self.create_for_connection_change(message, user, programming_group)
+    create!(
+      user:,
+      programming_group:,
+      study_group_id: user.current_study_group_id,
+      action: message[:action],
+      status: message[:status]
+    )
+  end
+
   def self.data_attribute(event, delta)
     event[:delta] = {data: delta} if delta.present?
-    event.presence if event.present? # TODO: As of now, we are storing the `current_user_id` most of the times. Intended?
+    event.presence if event.present? # TODO: As of now, we are storing the `session_id` most of the times. Intended?
```

Comment:
Should we remove the session id? I probably don't need it. Could only be used to check if multiple windows where used from a user.

Reply:
I'd go with it, and hence added df3ca1e4d802c12b2eea8135c570e1fe9629376b. Please give it a quick try; afterwards I'd proceed with merging this one.

Reply:
Looks good! 👍🏻